src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -174,16 +159,54 @@ export class DatasetFlowTableHelpers {
         return "";
     }
 
-    public static waitingBlockText(startCondition: MaybeNull<FlowStartCondition>, datasetKind: DatasetKind): string {
+    public static durationBlockText(node: FlowSummaryDataFragment): string {
+        const descriptionTypename = node.description.__typename;
+        switch (descriptionTypename) {
+            case "FlowDescriptionDatasetPollingIngest":
+            case "FlowDescriptionDatasetPushIngest": {
+                const startCondition = node.startCondition as FlowStartConditionSchedule;
+                return `wake up time: ${moment(startCondition.wakeUpAt).format(AppValues.CRON_EXPRESSION_DATE_FORMAT)}`;
+            }
+            case "FlowDescriptionDatasetExecuteTransform": {
+                const startCondition = node.startCondition as FlowStartConditionBatching;
+                switch (node.startCondition?.__typename) {
+                    case "FlowStartConditionExecutor":
+                        return `await since: ${moment(node.timing.awaitingExecutorSince ?? "").format(
+                            AppValues.CRON_EXPRESSION_DATE_FORMAT,
+                        )}`;
+                    case "FlowStartConditionSchedule": {
```

Comment:
Transform will never wait for schedule. Possible states: batching, throttling, executor